tests/sparseml/onnx/benchmark/test_info.py

```diff
@@ -38,7 +34,9 @@
 @pytest.fixture(scope="module")
 def mobilenet_fixture() -> Model:
     with tempfile.TemporaryDirectory() as onnx_dir:
-        yield Zoo.download_model_from_stub(TEST_STUB, override_parent_path=onnx_dir)
+        model = Model(TEST_STUB)
+        model.path = onnx_dir
+        yield model
```

Comment:
we should trigger the download here in the fixture if we don't already

Reply:
Let me look into this in more detail.